packages/react/testing-library/etc/react-lynx-testing-library.api.md

```diff
@@ -7,9 +7,9 @@
 import { BoundFunction } from '@testing-library/dom';
 import { ComponentChild } from 'preact';
 import { ComponentType } from 'preact';
-import { ElementTree } from '@lynx-js/test-environment';
-import { LynxElement } from '@lynx-js/test-environment';
-import { LynxEnv } from '@lynx-js/test-environment';
+import { ElementTree } from '@lynx-js/testing-environment';
+import { LynxElement } from '@lynx-js/testing-environment';
+import { LynxTestingEnvironment } from '@lynx-js/testing-environment';
```

Comment:
I don't mind you calling this `LynxTestingEnv` if you'd like this to be shorter

Reply:
Thanks, I have renamed to `LynxTestingEnv`